controllers/machine_controller_noderef.go

```diff
@@ -129,4 +125,4 @@ func (r *MachineReconciler) getNodeReference(client corev1.NodesGetter, provider
 	}
 
 	return nil, ErrNodeNotFound
-}
+}
```

Comment:
Please don't remove the trailing newline. In Goland, prefs - editor - general - ensure line feed at file end on save